astro/tests/integration_test.go

```diff
@@ -95,7 +108,11 @@ func TestProjectPlanError(t *testing.T) {
 		t.Run(version, func(t *testing.T) {
 			result := RunTest(t, []string{"plan"}, "fixtures/plan-error", version)
 			assert.Contains(t, result.Stderr.String(), "foo: [31mERROR")
-			assert.Contains(t, result.Stderr.String(), "Error parsing")
+			errorMessage := "Error parsing"
+			if stringVersionMatches(version, ">=0.12") {
+				errorMessage = "Argument or block definition required"
```

Comment:
This string has a weird character at the end (character code 27). I'd probably just remove it.